opt_out_http_api/store/memory.py

```diff
@@ -21,7 +21,18 @@ class OptOutMemory(object):
 
     def __init__(self):
         # _store maps (address_type, address) pairs to opt outs
-        self._store = {}
+        self._store = {
```

Comment:
This is a bad change. The store should start empty, not with random data in it. If a particular test needs specific data to be present, the test itself should add that data to the store.